mat64/symmetric.go

```diff
@@ -237,7 +236,30 @@ func (s *SymDense) SymRankK(a Symmetric, alpha float64, x Matrix) {
 //  s = x * x'
 func (s *SymDense) SymOuterK(x Matrix) {
 	r, _ := x.Dims()
-	s.reuseAs(r)
+
```

Comment:
Interesting. I didn't actually implement it to verify. Would you like me to take a stab?